{"uid":"1991827d62d272db","name":"Check App 1 built and running","fullName":"Check App 1 built and running","historyId":"1767beb10f129ee6bf3835c8bd380096","time":{"start":1698784518000,"stop":1698784518003,"duration":3},"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress.\n\n  > Cannot find module './src/components/Button.js'\nwhile loading \"./Button\" from webpack/container/reference/app2\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"Error: The following error originated from your application code, not from Cypress.\n\n  > Cannot find module './src/components/Button.js'\nwhile loading \"./Button\" from webpack/container/reference/app2\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at ./Button (http://localhost:3002/static/js/remoteEntry.js:1050:11)\n    at get (http://localhost:3002/static/js/remoteEntry.js:1057:23)\n    at handleFunction (http://localhost:3001/static/js/builder-runtime.js:274:31)\n    at onInitialized (http://localhost:3001/static/js/builder-runtime.js:286:60)\n    at handleFunction (http://localhost:3001/static/js/builder-runtime.js:279:25)\n    at onExternal (http://localhost:3001/static/js/builder-runtime.js:285:68)\n    at <unknown> (http://localhost:3001/static/js/builder-runtime.js:276:52)\n    at async Promise.all (index 0)\n    at async Promise.all (index 1)","flaky":true,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress.\n\n  > Cannot find module './src/components/Button.js'\nwhile loading \"./Button\" from webpack/container/reference/app2\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"Error: The following error originated from your application code, not from Cypress.\n\n  > Cannot find module './src/components/Button.js'\nwhile loading \"./Button\" from webpack/container/reference/app2\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at ./Button (http://localhost:3002/static/js/remoteEntry.js:1050:11)\n    at get (http://localhost:3002/static/js/remoteEntry.js:1057:23)\n    at handleFunction (http://localhost:3001/static/js/builder-runtime.js:274:31)\n    at onInitialized (http://localhost:3001/static/js/builder-runtime.js:286:60)\n    at handleFunction (http://localhost:3001/static/js/builder-runtime.js:279:25)\n    at onExternal (http://localhost:3001/static/js/builder-runtime.js:285:68)\n    at <unknown> (http://localhost:3001/static/js/builder-runtime.js:276:52)\n    at async Promise.all (index 0)\n    at async Promise.all (index 1)","steps":[],"attachments":[{"uid":"61c2fffa457a57c6","name":"e2e/checkApplications.cy.ts:2023-10-31T20:35:11.671Z","source":"61c2fffa457a57c6.png","type":"image/png","size":189728},{"uid":"8cc0b58b56a2f27b","name":"video recording","source":"8cc0b58b56a2f27b.mp4","type":"video/mp4","size":2706575}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":2,"hasContent":true},"afterStages":[],"labels":[{"name":"package","value":"e2e.checkApplications.cy.ts"},{"name":"parentSuite","value":"Basic Host remote"},{"name":"suite","value":"Check App 1"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":13,"broken":0,"skipped":0,"passed":48,"unknown":0,"total":61},"items":[{"uid":"ae61c654bc0d4c6f","reportUrl":"https://module-federation.github.io/module-federation-examples/1510//#testresult/ae61c654bc0d4c6f","status":"passed","time":{"start":1698739610609,"stop":1698739610611,"duration":2}},{"uid":"f6ba9980ae654fe5","reportUrl":"https://module-federation.github.io/module-federation-examples/1509//#testresult/f6ba9980ae654fe5","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress.\n\n  > Cannot find module './src/components/Button.js'\nwhile loading \"./Button\" from webpack/container/reference/app2\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1698741043629,"stop":1698741043631,"duration":2}},{"uid":"1bb8a69c5098f19","reportUrl":"https://module-federation.github.io/module-federation-examples/1504//#testresult/1bb8a69c5098f19","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress.\n\n  > Cannot find module './src/components/Button.js'\nwhile loading \"./Button\" from webpack/container/reference/app2\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1698737888131,"stop":1698737888133,"duration":2}},{"uid":"4da03f0ab23f91b5","reportUrl":"https://module-federation.github.io/module-federation-examples/1502//#testresult/4da03f0ab23f91b5","status":"passed","time":{"start":1698734698387,"stop":1698734698389,"duration":2}},{"uid":"c3116c55293f18c3","reportUrl":"https://module-federation.github.io/module-federation-examples/1501//#testresult/c3116c55293f18c3","status":"passed","time":{"start":1698732352228,"stop":1698732352230,"duration":2}},{"uid":"e3679c5655599ed7","reportUrl":"https://module-federation.github.io/module-federation-examples/1499//#testresult/e3679c5655599ed7","status":"passed","time":{"start":1698726854453,"stop":1698726854454,"duration":1}},{"uid":"394cc3873555e749","reportUrl":"https://module-federation.github.io/module-federation-examples/1497//#testresult/394cc3873555e749","status":"passed","time":{"start":1698724558714,"stop":1698724558716,"duration":2}},{"uid":"43c31704fc3bb401","reportUrl":"https://module-federation.github.io/module-federation-examples/1495//#testresult/43c31704fc3bb401","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress.\n\n  > Cannot find module './src/components/Button.js'\nwhile loading \"./Button\" from webpack/container/reference/app2\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1698725252905,"stop":1698725252907,"duration":2}},{"uid":"2ab0a890d4ae09ce","reportUrl":"https://module-federation.github.io/module-federation-examples/1493//#testresult/2ab0a890d4ae09ce","status":"passed","time":{"start":1698723404872,"stop":1698723404874,"duration":2}},{"uid":"153ba5f6fa86fa46","reportUrl":"https://module-federation.github.io/module-federation-examples/1490//#testresult/153ba5f6fa86fa46","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress.\n\n  > Cannot find module './src/components/Button.js'\nwhile loading \"./Button\" from webpack/container/reference/app2\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1698721232302,"stop":1698721232304,"duration":2}},{"uid":"c2114788a3aaca66","reportUrl":"https://module-federation.github.io/module-federation-examples/1481//#testresult/c2114788a3aaca66","status":"passed","time":{"start":1698712940673,"stop":1698712940676,"duration":3}},{"uid":"96a48958c3a4308c","reportUrl":"https://module-federation.github.io/module-federation-examples/1479//#testresult/96a48958c3a4308c","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress.\n\n  > Cannot find module './src/components/Button.js'\nwhile loading \"./Button\" from webpack/container/reference/app2\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1698714080799,"stop":1698714080801,"duration":2}},{"uid":"f95b1b1d8df1b4cc","reportUrl":"https://module-federation.github.io/module-federation-examples/1477//#testresult/f95b1b1d8df1b4cc","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress.\n\n  > Cannot find module './src/components/Button.js'\nwhile loading \"./Button\" from webpack/container/reference/app2\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1698713861994,"stop":1698713861997,"duration":3}},{"uid":"303e001babd9e6c0","reportUrl":"https://module-federation.github.io/module-federation-examples/1470//#testresult/303e001babd9e6c0","status":"passed","time":{"start":1698706083669,"stop":1698706083671,"duration":2}},{"uid":"3bdc2159a4113c37","reportUrl":"https://module-federation.github.io/module-federation-examples/1468//#testresult/3bdc2159a4113c37","status":"passed","time":{"start":1698702463707,"stop":1698702463710,"duration":3}},{"uid":"4908b7978ca2fe","reportUrl":"https://module-federation.github.io/module-federation-examples/1467//#testresult/4908b7978ca2fe","status":"passed","time":{"start":1698696893194,"stop":1698696893197,"duration":3}},{"uid":"8362bcf606987487","reportUrl":"https://module-federation.github.io/module-federation-examples/1466//#testresult/8362bcf606987487","status":"passed","time":{"start":1698691889394,"stop":1698691889397,"duration":3}},{"uid":"bd7c76be06081e0","reportUrl":"https://module-federation.github.io/module-federation-examples/1459//#testresult/bd7c76be06081e0","status":"passed","time":{"start":1698655936353,"stop":1698655936355,"duration":2}},{"uid":"eecf30af3f7eda1a","reportUrl":"https://module-federation.github.io/module-federation-examples/1458//#testresult/eecf30af3f7eda1a","status":"passed","time":{"start":1698652087813,"stop":1698652087814,"duration":1}},{"uid":"af9617c4868df109","reportUrl":"https://module-federation.github.io/module-federation-examples/1454//#testresult/af9617c4868df109","status":"passed","time":{"start":1698646005177,"stop":1698646005179,"duration":2}}]},"tags":[]},"source":"1991827d62d272db.json","parameterValues":[]}